backend/src/main/java/codesquad/airdnb/global/util/RandomStringUtil.java

```diff
@@ -0,0 +1,22 @@
+package codesquad.airdnb.global.util;
+
+import java.security.SecureRandom;
+import java.util.Base64;
+
+public class RandomStringUtil {
+
+    private static final SecureRandom secureRandom = new SecureRandom();
+    private static final Base64.Encoder base64Encoder = Base64.getUrlEncoder().withoutPadding();
+
+    private RandomStringUtil() {
+        throw new UnsupportedOperationException("유틸리티 클래스의 인스턴스를 생성할 수 없습니다.");
+    }
+
+    public static String generateRandomPassword() {
+        int length = 20;
+        byte[] randomBytes = new byte[20];
+        secureRandom.nextBytes(randomBytes);
+        String encodedString = base64Encoder.encodeToString(randomBytes);
+        return encodedString.length() > length ? encodedString.substring(0, length) : encodedString;
+    }
```

Comment:
```suggestion
    private static final Integer DEFAULT_PASSWORD_LENGTH = 20;

    public static String generateRandomPassword() {
        byte[] randomBytes = new byte[DEFAULT_PASSWORD_LENGTH];
        secureRandom.nextBytes(randomBytes);
        String encodedString = base64Encoder.encodeToString(randomBytes);
        return encodedString.substring(0, Math.min(DEFAULT_PASSWORD_LENGTH, encodedString.length()));
    }
```

- length 가 항상 20글자 일까요?
- password 를 만드는 행위가 member 말고 또 일어날까요?
- util 함수의 존재 가치는 무엇일까요? with OOP

Reply:
'랜덤문자열생성' 도 아니고, 랜덤 비밀번호 생성이라는 목적을 가진 메소드이기 때문에 global에 해당하는 유틸클래스보다 Member 도메인에 포함되는 것이 더 좋은 방향이 맞는 것 같습니다.

또한 상수화되지 않은 숫자를 이용하는 것도 좋지 않은 방법같네요